src/vm/amd64/JitHelpers_Fast.asm

```diff
@@ -955,5 +955,29 @@ endif ; _DEBUG
 
 NESTED_END TailCallHelperStub, _TEXT
 
-        end
+PAGE_SIZE equ 1000h
+
+LEAF_ENTRY JIT_StackProbe, _TEXT
```

Comment:
I'd like to see a line or two of comments in these asm helpers describing the purpose and function of the helper (what it does), in addition to the register "on entry" / "on exit" documentation (which is super useful).

It would also be useful to indicate what all the requirements are around each helper (as the requirements differ per platform). E.g., on Linux you can't probe beyond ESP/RSP.

Reply:
@BruceForstall I believe I have addressed all your suggestions - please take a look and let me know if I need to clarify anything else